lib/challenge.js

```diff
@@ -20,6 +20,21 @@
 
 // Answer Starts Here
 
+  /*
+
+  'Calendar'
+  Description: Our app helps users manage their time.
+  Every 'calendar' has
+  - User
+  - Days
+  - Weeks
+  Every 'calendar' can
+  - Keep track of appointments and events
+  - Tell the user what day it is
+  - Tell the user how long until ba certain event
+
+  */
+
```

Comment:
I think you may have mis-read the instructions; as written here, 'calendar' is one abstraction with three properties ('user', 'days', and 'weeks'), rather than three distinct abstractions.